dart-if/include/dash/dart/if/dart_types.h

```diff
@@ -328,7 +328,13 @@ typedef enum
 dart_locality_scope_t;
 
 /** Maximum size of a host name string in \ref dart_hwinfo_t */
-#define DART_LOCALITY_HOST_MAX_SIZE        (HOST_NAME_MAX)
+#if defined(HOST_NAME_MAX)
+  #define DART_LOCALITY_HOST_MAX_SIZE      (HOST_NAME_MAX)
```

Comment:
In this manpage it looks like we need to include `unistd.h`: https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/gethostname.3.html

Maybe that fixes it?